lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -427,7 +427,7 @@ def add_managed_billing(xml, options)
 
       def parse(body)
         response = {}
-        xml = REXML::Document.new(body)
+        xml = REXML::Document.new(body.encoding.name == "UTF-8" ? body : body.encode("UTF-8", "ISO-8859-1"))
```

Comment:
Are you certain `ISO-8859-1` is always the other encoding? Why not just `body.encode("UTF-8")`?